ci_framework/roles/edpm_build_images/tasks/main.yml

```diff
@@ -36,6 +36,10 @@
   retries: 60
   delay: 10
 
+- name: Add certs to the image
+  when: cifmw_edpm_build_images_cert_install_command is defined
```

Comment:
`cifmw_edpm_build_images_cert_install_command` doesn't seems a good name for that control var.
maybe just `cifmw_edpm_build_images_run_add_cert`

Reply:
I think it's still the wrong parameter name?